thespian/system/transport/TCPTransport.py

```diff
@@ -882,10 +881,8 @@ def _runWithExpiry(self, incomingHandler):
 
             wrecv = list(filter(None, wrecv))
             wsend = list(filter(None, wsend))
-            wrecv.extend(list(filter(None,
-                                     [I
-                                      for I in self._incomingSockets
-                                      if not self._incomingSockets[I].backoffPause()])))
+            wrecv.extend([ I for I in self._incomingSockets if I and
```

Comment:
Any idea why `list(filter(..))` ?  Isn't the result of `filter()` always a list?